.github/workflows/ci.yml

```diff
@@ -80,6 +80,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Setup nodejs
+        uses: actions/setup-node@v3
+      - name: Install packages
+        run: npm install ajv
+      - name: linter
+        run: node ./utils/reply_schema_linter.js
```

Comment:
maybe we wanna run this linter, or another validation in ci.yml (i.e. after moving the full test thing to daily.yaml)

Reply:
@fadidahanna last thing to do: fix GH workflows (conditional linter in CI, full validation in daily)